src/odemis/driver/tescan.py

```diff
@@ -48,11 +49,125 @@
 TESCAN_PXL_LIMIT = 100
 PROBE_CURRENT_RANGE = (10e-12, 100e-9)
 
+DEFAULT_ION_PRESET = ""
+
+
+class DeviceType(Enum):
+    ELECTRON = "electron"
+    ION = "ion"
+
+DEVICE_LABELS = {DeviceType.ELECTRON: "SEM", DeviceType.ION: "FIB"}
```

Comment:
I think you can drop this, and all the (2) places where it's used can just use the detector name.